apps/mobile/src/libs/components/white-background-wrapper/white-background-wrapper.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+import { View } from "~/libs/components/components";
+import { globalStyles } from "~/libs/styles/styles";
+
+import { styles } from "./styles";
+
+type Properties = {
+	children: React.ReactNode;
+};
+
+const WhiteBackgroundWrapper: React.FC<Properties> = ({ children }) => {
```

Comment:
Don't think that we actually need this, Can we just pass the `style` prop to ScreenWrapper?